docs/concepts/validation_decorator.md

```diff
@@ -39,151 +39,84 @@ except ValidationError as exc:
     """
 ```
 
-## Argument types
+## Parameter types
 
-Argument types are inferred from type annotations on the function, arguments without a type decorator are considered
-as `Any`. All types listed in [types](../concepts/types.md) can be validated, including Pydantic models and
-[custom types](../concepts/types.md#custom-types).
-As with the rest of Pydantic, types can be coerced by the decorator before they're passed to the actual function:
+Parameter types are inferred from type annotations on the function, or as [`Any`][typing.Any] if unannotated . All types listed in [types](types.md) can be validated, including Pydantic models and [custom types](types.md#custom-types).
```

Comment:
```suggestion
Parameter types are inferred from type annotations on the function, or as [`Any`][typing.Any] if not annotated. All types listed in [types](types.md) can be validated, including Pydantic models and [custom types](types.md#custom-types).
```

Reply:
Yeah I was unsure about this one. It is used in some places in the Python docs, but  I agree _not annotated_ seems better